packages/core/src/Hug.js

```diff
@@ -24,28 +22,25 @@ const BorderConcealer = styled(Box)`
   }
 `
 
-const HideableIcon = styled(Icon)`
-  ${display};
-`
+const Hug = ({ bg, color, p, fontSize, icon, iconDisplay, ...props }) => {
```

Comment:
What could iconDisplay be?

Reply:
Apparently that's a string for the CSS display property

Reply:
I feel like it's overkill to have that much power without doing something with css but I guess it's an existing thing 🤷‍♂